src/bidiMapper/domains/context/browsingContextProcessor.ts

```diff
@@ -230,10 +231,23 @@ export class BrowsingContextProcessor {
       }
     }
 
-    const result = await browserCdpClient.sendCommand('Target.createTarget', {
-      url: 'about:blank',
-      newWindow: params.type === 'window',
-    });
+    let result: Protocol.Target.CreateTargetResponse;
+
+    switch (params.type) {
+      case 'tab':
+        result = await browserCdpClient.sendCommand('Target.createTarget', {
+          url: 'about:blank',
+          newWindow: false,
+          browserContextId: referenceContext?.id ?? undefined,
```

Comment:
CDP `browserContextId` is not the same as BiDi's `browsingContext`.

Reply:
Should I revert this snippet? The spec says we should differentiate `tab` and `window` which we don't currently seem to do, but we should try to address it at another time.

Reply:
```
// TODO: Pass correct browserContextId.
// browserContextId: referenceContext?.id ?? undefined,
```

Would this comment be appropriate?

Reply:
In order to implement it properly, we need to:
1. Add `cdpBrowserContextId` to `CdpTarget`.
2. Use it: `browserContextId: referenceContext?.target.cdpBrowserContextId ?? undefined`

Reply:
Given your previous comment:

> Chrome doesn't support opening tab next to another one, so the parameter should be ignored. No need it TODO.

Omitting this for now.